tcms/core/management/commands/refresh_permissions.py

```diff
@@ -0,0 +1,41 @@
+from django.core.management import call_command
+from django.core.management.base import BaseCommand
+from tcms.utils.permissions import assign_default_group_permissions
+
+
+class Command(BaseCommand):
+    help = ('Refresh permissions for Tester group '
+            '(set by DEFAULT_GROUPS setting) and remove stale ones.'
+            )
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--noinput', '--no-input', action='store_false',
+            dest='interactive',
+            help='Automatic mode. Does not require user input',
+        )
+
+    def handle(self, *args, **kwargs):
+        output = None
+        verbosity = 0
+        if kwargs['verbosity']:
+            output = self.stdout
+            verbosity = 2
```

Comment:
you should assign the value of kwargs['verbosity'] here, no ? More precisely you have a hard-coded verbosity=2 for update_permissions but for remove_stale_contenttypes why hard-code it to 2? 

Maybe all of them should use the same level of verbosity and the user will specify whatever they want.